File: Platazi_Learn/POO/Python/UberPool.py
from Car import Car

class uberPool(Car):
    brand = str
    model = str

    def __init__(self, license, driver, brand, model):
        super().__init__(license, driver)
        self.brand = brand
        self.model = model

    def printDataUberPool(self):
        super().printDataCar()
        print("Model: ", self.model)
        print("Brand: ", self.brand, "\n")

File: Platazi_Learn/POO/Java/UberPool.java
package Java;
class UberPool extends Car {
    String brand;
    String model;
    
    public UberPool(String license, Account driver, String brand, String model) {
        super(license, driver);
        this.brand = brand;
        this.model = model;

    }

    @Override
    void printDataCar() {
        super.printDataCar();
        System.out.println("Brand: " + brand);
        System.out.println("Model: " + model);
    }

}